lib/utils/documentdb.ts

```diff
@@ -462,15 +462,23 @@ export function replaceDocument<T>(
   client: DocumentDb.DocumentClient,
   documentUri: IDocumentDbDocumentUri,
   document: T & DocumentDb.NewDocument,
-  partitionKey: string
+  partitionKey: string,
+  etag?: string
 ): Promise<Either<DocumentDb.QueryError, T & DocumentDb.RetrievedDocument>> {
   return new Promise(resolve => {
+    const requestOptions = fromNullable(etag)
```

Comment:
for some unknow reason the _etag field is marked as optional in the RetrievedDocument typings